### PR TITLE
Added pathplanner & misc util functions docs

### PR DESCRIPTION
**Pull request by Superbro525Alt**

### CODE REVIEW

**Review by spacey-sooty — CHANGES_REQUESTED** [review]
When adding subheadings just use function name not signature

**Review by spacey-sooty — CHANGES_REQUESTED** [review]
everything should be included using wombat.h remove that part of your instructions

**Review by spacey-sooty — CHANGES_REQUESTED** [review]
shorten util page name

**Comment by spacey-sooty** [comment]
No need for syntax highlighting in function name subheadings

**Comment by spacey-sooty** [comment]
After that will Aprove @Superbro525Alt

File: src/content/docs/reference/utils/utils.md [hunk 1]
```diff
@@ -0,0 +1,54 @@
+---
+title: 'Utility Functions'
+description: 'Miscellaneous utility functions'
+---
+
+This is a collection of utility functions that do not fit in one of the other catagories.
+
+## Usage
+
+### `now()`
```
Comment (spacey-sooty) [inline src/content/docs/reference/utils/utils.md]: Dont use highlighting for function names `### now`is how this should be done

File: src/content/docs/reference/utils/utils.md [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+
+Returns the current time in seconds since the program started.
+
+#### Usage
+    
+```cpp
+units::second_t time = now();
+```
+
+### `WritePose2NT`
```
Comment (spacey-sooty) [inline src/content/docs/reference/utils/utils.md]: same issue

File: src/content/docs/reference/utils/utils.md [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+### `WritePose3NT`
+
+Writes a Pose3d to a NetworkTable.
+
+#### Usage
+```cpp
+WritePose3NT(table, pose);
+```
+
+### `deadzone`
```
Comment (spacey-sooty) [inline src/content/docs/reference/utils/utils.md]: same issue

File: src/content/docs/reference/utils/voltage-controller.md [hunk 4]
```diff
@@ -0,0 +1,9 @@
+---
+title: 'Voltage Controller'
+description: 'Voltage controller for the robot'
+---
+
+A VoltageController is analagous to a MotorController, but in terms of voltage instead of speed.
+
+## Usage
```
Comment (spacey-sooty) [inline src/content/docs/reference/utils/voltage-controller.md]: This should be addressed before merge

File: src/content/docs/reference/utils/utils.md [hunk 5]
```diff
@@ -0,0 +1,54 @@
...
+
+Returns 0 if the value is within the deadzone, otherwise returns the value.
+
+#### Usage
+```cpp
+double val = deadzone(0.1, 0.05);
+```
+
+### `spow2`
```
Comment (spacey-sooty) [inline src/content/docs/reference/utils/utils.md]: same here

**Review by spacey-sooty — CHANGES_REQUESTED** [review]

**Comment by Superbro525Alt** [comment]
doing it now

**Comment by Superbro525Alt** [comment]
done

File: src/content/docs/reference/utils/voltage-controller.md (hunk 4 above)
Comment (Superbro525Alt) [reply src/content/docs/reference/utils/voltage-controller.md]: ok will do

Comment (Superbro525Alt) [reply src/content/docs/reference/utils/voltage-controller.md]: done

**Comment by Superbro525Alt** [comment]
i am finished

File: src/content/docs/reference/utils/Voltage Controller/motor-voltage-controller.md [hunk 6]
```diff
@@ -0,0 +1,69 @@
+---
+title: 'Motor Voltage Controller'
+description: 'Motor voltage controller for the robot'
+---
+The MotorVoltageController is an adapter for an `frc::MotorController` to a VoltageController.
+
+**See also: [Voltage Controller](../voltage-controller)**
```
Comment (spacey-sooty) [inline src/content/docs/reference/utils/Voltage Controller/motor-voltage-controller.md]: should be ../voltage-controller not ../

File: src/content/docs/reference/utils/Voltage Controller/voltage-controller.md [hunk 7]
```diff
@@ -0,0 +1,67 @@
+---
```
Comment (spacey-sooty) [inline src/content/docs/reference/utils/Voltage Controller/voltage-controller.md]: Rename this file to index.md (update all links to it as well)
This way the route for it is Voltage-Controller not voltage-controller/voltage-controller
Also remove the space in folder name

**Review by spacey-sooty — CHANGES_REQUESTED** [review]

File: src/content/docs/reference/utils/Voltage Controller/voltage-controller.md (hunk 7 above)
Comment (Superbro525Alt) [reply src/content/docs/reference/utils/Voltage Controller/voltage-controller.md]: done

File: src/content/docs/reference/utils/Voltage Controller/motor-voltage-controller.md (hunk 6 above)
Comment (Superbro525Alt) [reply src/content/docs/reference/utils/Voltage Controller/motor-voltage-controller.md]: done

File: src/content/docs/reference/utils/VoltageController/motor-voltage-controller.md [hunk 8]
```diff
@@ -0,0 +1,69 @@
+---
+title: 'Motor Voltage Controller'
+description: 'Motor voltage controller for the robot'
+---
+The MotorVoltageController is an adapter for an `frc::MotorController` to a VoltageController.
+
+**See also: [Voltage Controller](../../voltagecontroller)**
```
Comment (spacey-sooty) [inline src/content/docs/reference/utils/VoltageController/motor-voltage-controller.md]: [Voltage Controller](./index.md) Is what this should be

**Review by spacey-sooty — CHANGES_REQUESTED** [review]

File: src/content/docs/reference/utils/VoltageController/motor-voltage-controller.md (hunk 8 above)
Comment (Superbro525Alt) [reply src/content/docs/reference/utils/VoltageController/motor-voltage-controller.md]: what do you mean

Comment (spacey-sooty) [reply src/content/docs/reference/utils/VoltageController/motor-voltage-controller.md]: Sorry forgot abt markdown formatting working on GitHub I meant `[Voltage Controller](./index.md)` is what that should be @Superbro525Alt

**Review by spacey-sooty — APPROVED** [review]

**Pull request merged**